src/templates/cc-addon-encryption-at-rest-option.js

```diff
@@ -0,0 +1,25 @@
+import '../molecules/cc-error.js';
+import { html } from 'lit-element';
+import { assetUrl } from '../lib/asset-url.js';
+import { i18n } from '../lib/i18n.js';
+
+const LOGO = assetUrl(import.meta, '../assets/encryption-at-rest.svg');
```

Comment:
The current convention is `camelCaseNameOfTheImageSvg` with a `Svg` suffix.

Reply:
Should be fixed now